### ISSUE TITLE
Give user expand/shrink default view setting in admin panel.

### ISSUE THREAD

**Issue by Duoquote** [django-treebeard/django-treebeard#218]
I think the issue is pretty self explanatory, I don't want all the items to come expanded, even if it's cropped and continues on other pages, I can change the page's max listing to 10000 or infinite to display them all but, by default, making it come shrinked would help a lot.

Basically;
I don't want this:
![image](https://user-images.githubusercontent.com/16341937/110642325-fa20ee00-81c3-11eb-93b1-c2be8d049f94.png)

I want this:
![image](https://user-images.githubusercontent.com/16341937/110642379-073ddd00-81c4-11eb-80c4-ca800df5d002.png)

to come by default.

I feel so rude saying that `want` word and I am sorry to say that, I don't know what word to use instead of that but what I mean is, being able to control this basic functionality, in any way, would be great in many cases.

If there is any js hack I can do or something like that, I am open for that too.

Thanks for this good work and keeping it up!

**Comment by drazenzen** [comment]
This small js should do the trick of collapsing all nodes on some level (on level 2 in given example).
It is not perfect since it will call click method on selected nodes. Visual collapses will appear on page load.
```javascript
(function($) {
    $(window).on('load', function() {
        console.log("Loaded");
        var rows = $("[level=2]");
        var btns = rows.children().children("a.collapse");
        for (var i = 0; i < btns.length; i++) {
            btns[i].click();
        }
        console.log("Done");
    });
})(django.jQuery);
```
Hook it up inside admin class, e.g.
```python
class ModelAdmin(TreeAdmin):
    class Media:
        js = ('admin/js/jquery.init.js', "path/to/script/above/ext.js")
```
and tree nodes, on admin page load, should be collapsed to level 2

Tested on:
Django=3.2.7
django-treebeard==4.3

**Comment by DesignsToGrow** [comment]
Yes this makes a lot of sense. I'm dealing with a project that has almost 200 categories so it would be really great if you could collapse the categories by default.

**Comment by solarissmoke** [comment]
Took a while, but I have just released v7 of Treebeard with a fresh implementation of the admin integration. Trees are loaded incrementally, starting only with the root nodes, and fetching children when a node is expanded. The queries are much more efficient and can scale to any size of tree now.
